src/geowombat/backends/xarray_.py

```diff
@@ -300,7 +282,9 @@ def warp_netcdf_vars():
 
         yield xr.concat(
             (
-                open_rasterio(wobj, **kwargs).assign_coords(
+                open_rasterio(
+                    wobj, nodata=ref_kwargs['nodata'], **kwargs

```

Comment:
Passing `nodata` to `open_rasterio` so that nodata values can override file opening if the user requests.